pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -93,7 +95,8 @@ buildPythonPackage {
   build-system = [
     distutils
     wheel
-  ] ++ lib.optionals cudaSupport [ addDriverRunpath ];
+  ] ++ lib.optionals cudaSupport ([ addDriverRunpath ]
+    ++ lib.optionals cudaPackages.cudaFlags.isJetsonBuild [ cudaPackages.autoAddCudaCompatRunpath ]);
```

Comment:
nixfmt-rfc-style, but also maybe split into two parallel branches `cudaSupport` and `cudaSupport && isJetsonBuild` two avoid extra indent

Reply:
Done